### PR TITLE
Add Blaise

### PR DESCRIPTION
**Pull request by nicolaslt**
Moving this here instead of my team's private repo :smiley: 

I think the README in the PR will explain `Blaise` better than I would in the PR description.

### CODE REVIEW

File: packages/blaise/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import avroToJson from '@ovotech/avro-mock-generator';
+import {
+  CastleEachBatchPayload,
+  CastleEachMessagePayload,
+} from '@ovotech/castle';
+import merge = require('lodash.merge');
+import { DeepPartial } from 'ts-essentials';
+import { BlaiseDefaults, IBlaise } from './types';
+
+// tslint:disable-next-line
```
Comment (ivank) [inline packages/blaise/src/index.ts]: Can we use eslint, as that's what the other packages here are using and it is, you know ["the future"](https://medium.com/palantir/tslint-in-2019-1a144c2317a9).  Also might be worth disabling the specific error not linting altogether

File: packages/blaise/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+const coffee = withAvro({ avro: { pickUnion: ['coffee'] } });
+const tea = withAvro({ avro: { pickUnion: ['tea'] } });
+
+coffee.getDefault(); // { avro: { schema, pickUnion: ['coffee']}}
+tea.getDefault(); // { avro: { schema, pickUnion: ['tea']}}
+```
+
+### Generate an Avro Message
+
+```typescript
```
[the remark this answers is not in the document]
Comment (nicolaslt) [reply packages/blaise/README.md]: This is Amazing.

**Comment by nicolaslt** [comment]
@ivank All comments addressed.

**Comment by nicolaslt** [comment]
Good point, I'll do that now.

**Pull request merged**